api/proto/teleport/legacy/types/types.proto

```diff
@@ -6722,6 +6722,19 @@ message PluginOktaSyncSettings {
   // DisableBidirectionalSync prevents syncing anything from Teleport to Okta. I.e. prevents
   // creating Okta assignments.
   bool disable_bidirectional_sync = 10;
+
+  // UserSyncSource defines the source of truth for Okta users. It can be either users assigned to
+  // the connector SAML application or users in the whole Okta organization (legacy).
+  UserSyncSource user_sync_source = 11;
+
+  enum UserSyncSource {
```

Comment:
Is this used anywhere where users might see it in JSON form? (audit events, config, etc)

The marshaling for proto enums isn't great, so I wonder if it would be better to just make it a string.

Reply:
Yeah, events and backend. I'll change it.